### PR TITLE
docs/Homebrew-on-Linux: Centre the logo

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

**Comment by jonchang** [comment]
Maybe we can fix the sponsor badges as well? It might be worthwhile to add some or an `_include` to https://github.com/Homebrew/brew.sh for prettier images.

**Comment by sjackman** [comment]
> Maybe we can fix the sponsor badges as well?

Done.

File: docs/Homebrew-on-Linux.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ redirect_from:
   - /Linux
   - /Linuxbrew
 ---
+
+<p style="text-align:center">
```
Comment (MikeMcQuaid) [inline docs/Homebrew-on-Linux.md]: I think at the point we're adding this to the Markdown it means we should instead add/modify CSS on brew.sh as @jonchang suggests. Shout if you need help with that.

Comment (sjackman) [reply docs/Homebrew-on-Linux.md]: I've opened PR https://github.com/Homebrew/brew.sh/pull/342 to modify the style sheet.

**Comment by MikeMcQuaid** [comment]
Closing in favour of
https://github.com/Homebrew/brew.sh/pull/342

**Comment by sjackman** [comment]
The sizes of the images should be fixed by PR https://github.com/Homebrew/brew.sh/pull/342, but it doesn't centre the images, which was the original purpose of this PR.

**Comment by sjackman** [comment]
For some unknown reason I can't reopen this PR, so I've opened PR https://github.com/Homebrew/brew/pull/5886

**Pull request closed without merging**